spring-cloud-gcp-examples/spring-cloud-gcp-integration-pubsub-example/spring-cloud-gcp-integration-pubsub-example-receiver/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+server.port=8082
```

Comment:
You don't have Web in this app any more. Therefore no any `server.port` properties.
I would say no `application.properties` at all!